lighthouse-cli/test/fixtures/byte-efficiency/tester.html

```diff
@@ -90,41 +91,63 @@ <h2>Byte efficiency tester page</h2>
   <!-- FAIL(responsive): image is 50% used at DPR 3 -->
   <!-- PASS(responsive-inverse): image is big enough -->
   <!-- PASS(offscreen): image is onscreen -->
+  <!-- PASS(unsized-images): has CSS sizing -->
   <img style="width: 256px; height: 170px;" src="lighthouse-1024x680.jpg">
 
   <!-- PASS(optimized): image is JPEG optimized -->
   <!-- PASS(webp): image is WebP optimized -->
   <!-- FAIL(responsive): image is 25% used at DPR 2.625 -->
   <!-- PASS(responsive-inverse): image is big enough -->
   <!-- PASS(offscreen): image is offscreen but lazily loaded -->
+  <!-- PASS(unsized-images): CSS sizing is valid with current checks -->
   <img style="width: 0px; height: 0px;" src="lighthouse-2048x1356.webp?size0" loading="lazy">
 
   <!-- PASS(optimized): image is JPEG optimized -->
   <!-- FAIL(webp): image is not WebP optimized -->
   <!-- FAIL(responsive): image is not fully used at DPR 2.625 -->
   <!-- PASS(responsive-inverse): image is big enough -->
   <!-- PASS(offscreen): image is onscreen -->
+  <!-- PASS(unsized-images): has CSS sizing -->
   <img style="width: 160px; height: 110px;" src="lighthouse-480x320.jpg">
 
+  <!-- PASS(optimized): image is JPEG optimized -->
+  <!-- FAIL(webp): image is not WebP optimized -->
+  <!-- FAIL(responsive): image is not fully used at DPR 2.625 -->
+  <!-- PASS(responsive-inverse): image is big enough -->
+  <!-- PASS(offscreen): image is onscreen -->
+  <!-- PASS(unsized-images): has attribute sizing -->
+  <img width="160" height="110" src="lighthouse-480x320.jpg">
```

Comment:
```suggestion
  <img width="160" height="110" src="lighthouse-480x320.jpg?attributesized">
```

it's helpful to make each image name unique so we can tell them apart in the expectations when things fail :)

Reply:
Sounds great, I agree! To be clear, this means I should also copy + rename the images & insert them into `lighthouse-cli/test/fixtures/byte-efficiency/`?

Reply:
Oh no need to make copies of the files! Just adding a query parameter like the suggestion does is sufficient to differentiate the network requests to our audits :)